libs/map/src/components/legend-item/vl-map-legend-item.ts

```diff
@@ -0,0 +1,51 @@
+import { html, PropertyDeclarations, TemplateResult, unsafeCSS, css } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { BaseLitElement } from '@domg-wc/common-utilities';
+import legendStyles from '../legend/vl-map-legend.uig-css';
+import styles from './vl-map-legend-item.uig-css';
+
+@customElement('vl-map-legend-item')
+export class VlMapLegendItem extends BaseLitElement {
+    icon = null;
+    label = '';
+    layer = '';
+
+    static get styles() {
```

Comment:
is het mogelijk dit veranderen zodat de statische styles hier worden ingesteld conform hoe we het standaard doen voor Lit componenten

bv.:

```ts
static get styles(): (CSSResult | CSSResult[])[] {
        return [legendStyles, styles];
    }
```

of mss er een specifieke reden dit anders te doen